client/src/views/FollowList/FollowList.vue

```diff
@@ -19,9 +15,27 @@ export default {
   },
   props: {
     followList: Array
+  },
+  data () {
+    return {
+      followSearch: ''
+    }
+  },
+  computed: {
+    filteredFollowList () {
+      return this.followList.filter(post => {
+        return post.nickname.toLowerCase().includes(this.followSearch.toLowerCase())
```

Comment:
For future try using object destructuring aka post nickname could be _({nickname = ''}) => nickname.toLowerCase()_... (simply more readable)
Also applying _this.followSearch.toLowerCase()_ on every post iteration increases computing time (even though V8 has optimization trick for this - still try preventing writing like this), you could have:
_const searchToken = this.followSearch.toLowerCase();_ before running the loop;